### ISSUE TITLE
Connection timed out at client side

### ISSUE THREAD

**Issue by Spider84** [walkor/phpsocket.io#216]
I wrote simple socket.io server and start it in DEBUG mode with Single Worker instance.
Connecting to this server from WEB pages JS socket.io clients (5-10 clients at same time). All work fine if clients connecting with some intervar (minutes between connections).
If all clients do connect in same time (~1-5 second between connections) server generate connection event but clients will disconnects with _Connecting timedout_ error.
Look like server work fine (not freeze, process data from SQL fine) but not accepting clients.
After this I block clients by firewall, restart Workerman Daemon and then unblock clients by one.
How to debug this issue? 
HELP!! :)

**Comment by marcosmarcolin** [comment]
Closed due to inactivity or open for too long.

If you still need support, please open a new issue.
